code/modules/events/operative.dm

```diff
@@ -3,6 +3,7 @@
 	typepath = /datum/round_event/ghost_role/operative
 	weight = 0 //its weight is relative to how much stationary and neglected the nuke disk is. See nuclearbomb.dm. Shouldn't be dynamic hijackable.
 	max_occurrences = 1
```

Comment:
Instead of doing these changes, move this to event_oveeride.dm and set it's max occurrences to 0

Reply:
Allow it to increase weight uselessly.